### ISSUE TITLE
earpieceOrSpeakersToggle is missing in the new version of the audioplayers

### ISSUE THREAD

**Issue by dariocavada** [bluefireteam/audioplayers#1240]
## Checklist
- [X] I made sure that the feature I am proposing doesn't already exist
- [X] I've tried the forceSpeaker and respectSilence configuration (AudioContextConfig) with no success

## Use case / Problem

In the version 0.20 there was an `earpieceOrSpeakersToggle` method. 
I use this method for an audio guide, when you can decide if listed the audio with earpiece or if there are not many people around you use the speaker.

## Proposal / Solution
Add the routing capabilities to the new `AudioContextConfig` if it's possible or reintroduce the `earpieceOrSpeakersToggle` method

**Comment by Gustl22** [comment]
I think you have to set [AudioContext](https://github.com/bluefireteam/audioplayers/blob/2cb51b3071676cdb0d2211bec201fc5b05314691/packages/audioplayers_platform_interface/lib/api/audio_context_config.dart#L173) for each platform for yourself.

See example: https://github.com/bluefireteam/audioplayers/blob/2cb51b3071676cdb0d2211bec201fc5b05314691/packages/audioplayers/example/lib/tabs/audio_context.dart#L104

**Comment by yoer** [comment]
Speaker

> I think you have to set [AudioContext](https://github.com/bluefireteam/audioplayers/blob/2cb51b3071676cdb0d2211bec201fc5b05314691/packages/audioplayers_platform_interface/lib/api/audio_context_config.dart#L173) for each platform for yourself.
> 
> See example:
> 
> https://github.com/bluefireteam/audioplayers/blob/2cb51b3071676cdb0d2211bec201fc5b05314691/packages/audioplayers/example/lib/tabs/audio_context.dart#L104

force to speaker, is not success .
just run the latest example, also no success

**Comment by Gustl22** [comment]
Setting to earpiece will be available with #1388 like so:
```
AudioContextAndroid(
  isSpeakerphoneOn: false,
  usageType: AndroidUsageType.voiceCommunication,
  audioMode: AndroidAudioMode.inCommunication
)
```
